### PR TITLE
#2115 Change stream api prefix from stream to streams

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2115)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by saivirtue** [comment]
retry k8sConfigurator

File: docs/src/custom_streamapp.rst [hunk 1]
```diff
@@ -186,7 +186,7 @@ The above code does the following transformations:
 #. filter out that if ``filterName`` is null
 
    - here we get the value from **filterName** of definitions. the value you should update by
-     :ref:`Stream update api <rest-stream-update-information>`
+     :ref:`Stream update api <rest-streams-update-information>`
 
    PUT /v0/stream/XXX
```
Comment (vitojeng) [inline docs/src/custom_streamapp.rst]: Does this line need to update?
**PUT /v0/streams/XXX**

Comment (saivirtue) [reply docs/src/custom_streamapp.rst]: Yes. Good finding :)

**Comment by saivirtue** [comment]
failed IT tests are unrelated.

**Pull request merged**